tsconfig.json

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
What is this for? Also, TIL, there is a JSONC that supports comments in JSON. :)

Reply:
oops removed. I was testing and forgot to remove